cylc/flow/workflow_files.py

```diff
@@ -1353,7 +1353,7 @@ def install_workflow(
     cylc_install.mkdir(parents=True, exist_ok=True)
     if not source_link.exists():
         install_log.info(f"Creating symlink from {source_link}")
-        source_link.symlink_to(source)
+        source_link.symlink_to(source.resolve())
```

Comment:
Tested and this fixes the bug 👍

However, I'm wondering about the `if not source_link.exists()` line - if the source workflow is moved or deleted, then the same issue will resurface (as `broken_symlink.exists()` returns `False`, but `broken_symlink.symlink_to()` will fail).

I tried to install a different workflow with the same name, and it raises
```
$ cylc install -C cylc-forecasting-workflow/
WorkflowFilesError: Source directory between runs are not consistent.
```
so maybe a similar nicely-displayed error should be raised

Reply:
I'm struggling to reproduce this case, I suspect I have misunderstood. 
I have installed a workflow from `~/cylc-src/test-workflow` into `~/cylc-run/test-workflow/run1`
I then move `~/cylc-src/test-workflow` to `~/cylc-src/test-workflow2`. The symlink, as expected, breaks.
Then :
```
$pwd
~/cylc-src
$ cylc install -C test-workflow
WorkflowFilesError: no flow.cylc or suite.rc in test-workflow
```

I have also tried reinstalling:

```
$ cylc reinstall test-workflow/run1
WorkflowFilesError: Workflow source dir is not accessible: "~/cylc-src/test-workflow".
Restore the source or modify the ~/cylc-run/test-workflow/_cylc-install/source" symlink to continue. 
```
If I install the moved `test-workflow2`, it installs as a new flow, as expected, into  `~/cylc-run/test-workflow2/run1`

Reply:
Try:
- Create `~/cylc-src/test/flow.cylc`
- `$ cylc install test`
- Delete the `~/cylc-src/test/` dir
- Create `~/Documents/test/flow.cylc` (or wherever, doesn't have to be in Documents).
- `$ cylc install -C ~/Documents/test`

Reply:
Got it, thanks for clarifying. Should be fixed now with a test.